src/main/java/org/jabref/gui/exporter/ExportToClipboardAction.java

```diff
@@ -74,12 +78,13 @@ public void execute() {
                                           .orElse(null);
 
         Optional<Exporter> selectedExporter = dialogService.showChoiceDialogAndWait(Localization.lang("Export"), Localization.lang("Select export format"),
-                Localization.lang("Export"), defaultChoice, exporters);
+                                                                                    Localization.lang("Export"), defaultChoice, exporters);
 
         selectedExporter.ifPresent(exporter -> BackgroundTask.wrap(() -> exportToClipboard(exporter))
-                                                                .onSuccess(this::setContentToClipboard)
-                                                                .onFailure(ex -> { /* swallow as already logged */ })
-                                                                .executeWith(Globals.TASK_EXECUTOR));
+                                                             .onSuccess(this::setContentToClipboard)
+                                                             .onFailure(ex -> {
+                                                                 /* swallow as already logged */ })
```

Comment:
Instead of rethrowing the exception below, I would add logging + displaying to the user here.